src/main/java/com/aws/greengrass/mqtt/bridge/MQTTBridge.java

```diff
@@ -119,60 +123,66 @@ protected MQTTBridge(Topics topics, TopicMapping topicMapping, MessageBridge mes
 
     @Override
     public void install() {
```

Comment:
any reason not to synchronize the method instead of putting the method body in a lock?

Reply:
no reason other than naming, can move it to the method